playground/backend/internal/cloud_bucket/precompiled_objects_test.go

```diff
@@ -0,0 +1,218 @@
+	type args struct {
+		ctx         context.Context
+		examplePath string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *string
+		wantErr bool
+	}{
+		{
+			//try get output file from the bucket
+			name: "Test GetPrecompiledObjectOutput",
+			args: args{
+				ctx:         context.Background(),
+				examplePath: "SDK_JAVA/HelloWorld",
+			},
+			want:    &out,
+			wantErr: false,
+		},
+		{
+			//try get non-existent output file from the bucket
+			name: "Test FakeExampleOutput",
+			args: args{
+				ctx:         context.Background(),
+				examplePath: "SDK_JAVA/Fake",
+			},
+			want:    nil,
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			cd := &CloudStorage{}
+			got, err := cd.GetPrecompiledObjectOutput(tt.args.ctx, tt.args.examplePath)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("GetPrecompiledObjectOutput() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("GetPrecompiledObjectOutput() got = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestCloudStorage_GetPrecompiledObjects(t *testing.T) {
+	type args struct {
+		ctx      context.Context
+		sdk      pb.Sdk
+		category string
+	}
+	testExample := SdkToCategories{"SDK_JAVA": CategoryToPrecompiledObjects{"Common": PrecompiledObjects{ObjectInfo{
+		Name:        "HelloWorld",
+		CloudPath:   "SDK_JAVA/HelloWorld",
+		Description: "Description Hello World",
+		Type:        pb.PrecompiledObjectType_EXAMPLE,
+		Categories:  []string{"Common", "IO"},
+	}}}}
+	tests := []struct {
+		name    string
+		args    args
+		want    *SdkToCategories
+		wantErr bool
+	}{
+		{ //try get precompiled objects from the bucket with the "java" SDK and "Common" category
+			name: "Test GetPrecompiledObjects()",
```

Comment:
Do we have a negative test case here? If, for example, the SDK or category is wrong?